### PR TITLE
UI - search screen initial commit

### PR DESCRIPTION
**Pull request by bogdan-sava**

### CODE REVIEW

File: open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/init/DemoInMemoryInitializingBean.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/init/DemoInMemoryInitializingBean.java]: add copyright

File: open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/repository/ItemExecutorRepository.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/repository/ItemExecutorRepository.java]: add copyright in this & other files

Comment (bogdan-sava) [reply open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/repository/ItemExecutorRepository.java]: 👍

File: open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/init/DemoInMemoryInitializingBean.java (hunk 1 above)
Comment (bogdan-sava) [reply open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/init/DemoInMemoryInitializingBean.java]: 👍

**Comment by bogdan-sava** [comment]
recheck

**Pull request merged**